mobile/src/store/uam/reducer.ts

```diff
@@ -0,0 +1,31 @@
+import { createReducer } from '@reduxjs/toolkit';
+
+import { DataStatus } from '~/common/enums/enums';
+import { GroupsItemResponseDto } from '~/common/types/types';
+
+import { getGroups } from './actions';
+
+type State = {
+  dataStatus: DataStatus;
+  groups: GroupsItemResponseDto[];
```

Comment:
I think storing only `items` is not enough, add `total`, so `groups` will have a type `EntityPagination<GroupsItemResponseDto>`